build.sbt

```diff
@@ -7,10 +7,11 @@ import bijection._
 
 val buildLevelSettings = Seq(
   organization := "com.twitter",
-  crossScalaVersions := Seq("2.10.6", "2.11.8"),
+  crossScalaVersions := Seq("2.10.6", "2.11.8", "2.12.0"),
   scalaVersion := "2.11.8",
   javacOptions ++= Seq("-source", "1.6", "-target", "1.6"),
   javacOptions in doc := Seq("-source", "1.6"),
+  coverageEnabled := true,
```

Comment:
do we need to enable this in the build? Can't we do `+++$TRAVIS_SCALA_VERSION coverage test` or something?

What is it about this change that is requiring us to always do coverage?